### DISCUSSION TITLE
Result page - add histogram.

### DISCUSSION THREAD

**Discussion by victorKochkarev** [monkeytypegame/monkeytype#3519] [Ideas] [answered]
Adding a histogram on a result page might be a good idea. The histogram could represent a player's game results against all other game results within the same dictionary and time limit. 
I would be happy to add more details and work on the backed part of the project.

**Comment by Miodec** [comment, marked as answer]
Maybe not on the result page to not add clutter, but maybe I might add it to the account page.

**Reply by victorKochkarev** [reply]
On the account page it might be a good idea to show how the best result of a player against the best results of other players. While on result page it would make sense to show how the result of a game stands agents all the other games regardless if the game was the best or not.

In any case it might be worth to play around with it.
